test/answersWrapper.spec.js

```diff
@@ -0,0 +1,36 @@
+import { AnswersWrapper } from '../src/app/components/AnswersWrapper';
+import '@testing-library/jest-dom';
+import userEvent from '@testing-library/user-event';
+
+describe('Component should create and display answer buttons', () => {
+  it('When answerWrapper is created then proper buttons with answers are displayed', () => {
+    const answersWrapper = AnswersWrapper(['A', 'B'], 'B', jest.fn);
+    const [incorrect, correct] = answersWrapper.children;
+
+    expect(answersWrapper).toHaveClass('answersWrapper');
+
+    expect(incorrect.id).toBe('answer1');
+    expect(incorrect.innerText).toBe('A');
+    expect(incorrect).toHaveClass('button--answer');
+    expect(incorrect).toHaveClass('button--incorrectAnswer');
+    expect(incorrect.onClickFn).toBe(undefined);
+
+    expect(correct.id).toBe('answer2');
+    expect(correct.innerText).toBe('B');
+    expect(correct).toHaveClass('button--answer');
+    expect(correct).toHaveClass('button--correctAnswer');
+    expect(correct.onClickFn).toBe(undefined);
+  });
+
+  it('When answer is chosen then callback function should be called with proper parameters', () => {
```

Comment:
Ogólnie super zrobione i ładne testy! Rozbiłbym tylko ten przypadek na dwa. Bo gdyby ktos oposuł oznaczanie incorrect - to nie chcemy, żeby failował test, który sprawdza correct.

Reply:
Done